tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
+    local executable="$1"
+
+    distro_install_package dbus-x11
+
+    dbus-launch > dbus.env
+    init_dbus_env
```

Comment:
Each call to dbus-launch returns different values for `DBUS_SESSION_BUS_PID`, `DBUS_SESSION_BUS_WINDOWID` and `DBUS_SESSION_BUS_ADDRESS`, we need the same values used in the provider to be available to the consumer, saving them in dbus.env let's them to be used later at https://github.com/snapcore/snapd/pull/3484/files#diff-2b62efb5d1a59de96d357d3363649c78R37 or https://github.com/snapcore/snapd/pull/3484/files#diff-62fedbce031486f91580149e8fb17162R40